kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/firewall/PortForwardingTabUi.java

```diff
@@ -612,15 +653,17 @@ private void setModalFieldsHandlers() {
         // Set validations
         this.input.addBlurHandler(event -> {
             if (PortForwardingTabUi.this.input.getText().trim().isEmpty()
-                    || !PortForwardingTabUi.this.input.getText().trim().matches(FieldType.ALPHANUMERIC.getRegex())) {
+                    || !PortForwardingTabUi.this.input.getText().trim().matches(FieldType.ALPHANUMERIC.getRegex())
+                    || PortForwardingTabUi.this.input.getText().trim().length() > 15) {
```

Comment:
I would export the 15 as a class constant in order to define it and make it easy to change, if needed.